config/initializers/influxdb.rb

```diff
@@ -0,0 +1,24 @@
+InfluxDB::Rails.configure do |config|
+  config.influxdb_database = "miq_bot_#{Rails.env}"
```

Comment:
Can these values be pulled from Settings instead?  I'd prefer to have as much of the config in one place as possible.